knowledge-base/bflad/pr-size-labeler/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Pull Request size labeler'
+github-token:
+  action-input:
+    input: GITHUB_TOKEN
+    is-default: false
+  permissions:
+    pull-requests: write
+    pull-requests-reason: to add labels & comment on PRs #Checkout: https://github.com/bflad/pr-size-labeler/blob/master/src/github.sh#L15 
```

Comment:
We should also add the one this from created from. That is the main one: https://github.com/CodelyTV/pr-size-labeler